migration_steps/validation/api_tests/response_tests/app.py

```diff
@@ -18,105 +18,47 @@
-            },
-            {
-                "csv": "deputy_clients",
-                "identifier": "deputy"
-            },
+            {"csv": "deputies", "identifier": "deputy"},
+            {"csv": "deputy_clients", "identifier": "deputy"},
             {
                 "csv": "deputy_clients_count",
                 "identifier": "deputy",
-                "assert_on_count": True
-
-            },
-            {
-                "csv": "deputy_orders",
-                "identifier": "order_deputy"
-            },
-            {
-                "csv": "deputy_fee_payer",
-                "identifier": "order_deputy"
-            },
-        ],
-        "supervision_level": [
-            {
-                "csv": "supervision_level",
-                "identifier": "order"
+                "assert_on_count": True,
             },
+            {"csv": "deputy_orders", "identifier": "order_deputy"},
+            {"csv": "deputy_fee_payer", "identifier": "order_deputy"},
         ],
+        "supervision_level": [{"csv": "supervision_level", "identifier": "order"},],
         "death": [
-            {
-                "csv": "client_death_notifications",
-                "identifier": "client"
-            },
+            {"csv": "client_death_notifications", "identifier": "client"},
             {
                 "csv": "deputy_death_notifications",
                 "identifier": "deputy",
-                "no_retries": True
+                "no_retries": True,
             },
         ],
         "warnings": [
-            {
-                "csv": "warnings",
-                "identifier": "client",
-                "assert_on_list": True
-            },
-        ],
-        "crec": [
-            {
-                "csv": "crec",
-                "identifier": "client"
-            },
-        ],
-        "visits": [
-            {
-                "csv": "visits",
-                "identifier": "client"
-            },
-        ],
-        "reporting": [
-            {
-                "csv": "reports",
-                "identifier": "client"
-            },
+            {"csv": "client_warnings", "identifier": "client", "assert_on_list": True},
+            {"csv": "deputy_warnings", "identifier": "deputy", "assert_on_list": True},
         ],
+        "crec": [{"csv": "crec", "identifier": "client"},],
+        "visits": [{"csv": "visits", "identifier": "client"},],
+        "reporting": [{"csv": "reports", "identifier": "client"},],
         "invoice": [
-            {
-                "csv": "invoice",
-                "identifier": "client",
-                "assert_on_list": True
-            },
-        ],
-        "tasks": [
-            {
-                "csv": "tasks",
-                "identifier": "client",
-                "assert_on_list": True,
-                "list_in_field": "tasks"
-            },
+            {"csv": "invoice", "identifier": "client", "assert_on_list": True},
         ],
+        # "tasks": [
```

Comment:
Commented out tasks?

Reply:
yeah don't need to actually now that i have added logic where no file exists